lib/ansible/modules/network/nxos/nxos_pim_interface.py

```diff
@@ -350,12 +378,17 @@ def config_pim_interface(delta, existing, jp_bidir, isauth):
                 commands.append(command)
         command = None
 
+    if 'no ip pim sparse-mode' in commands:
+        # sparse is long-running on some platforms, process it last
+        commands.remove('no ip pim sparse-mode')
+        commands.append('no ip pim sparse-mode')
```

Comment:
Does this generate error code `400` when sent through nxapi?

Reply:
@chrisvanheuveln and I chatted about this.  No further changes needed here since we avoid the 400 error with the order change and the command is blocking.